contracts/Org.sol

```diff
@@ -111,19 +145,18 @@ contract Org is Administratable {
    * @param tokenAddress Address of desired token to query for balance
    * @return Balance of conract in token base unit of provided tokenAddress
    */
-
-  function getTokenBalance(address tokenAddress) external view returns (uint256) {
+  function getTokenBalance(address tokenAddress) public view returns (uint256) {
```

Comment:
Why is this public and not external now?

Reply:
Good catch! Missed this on the rebase. Fixed.